library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -2081,6 +2300,10 @@ rocblaslt_status runKernelFromInvocation(rocblaslt_handle       handle,
             {
                 logBenchFromTensileDataGemm(data->problem, data->inputs, true);
             }
+            else if(get_logger_layer_mode() & rocblaslt_layer_mode_log_profile)
```

Comment:
should use "if" instead of "else if"
HIPBLASLT_LOG_MASK is bitwise config. rocblaslt_layer_mode_log_bench and rocblaslt_layer_mode_log_profile should take effect at the same time if both are set.